package.json

```diff
@@ -7,34 +7,28 @@
     "build": "webpack --env production --config webpack.dist.js",
     "build-example": "webpack --env production",
     "build-e2e": "webpack --env production --config webpack.e2e.js",
-    "start": "webpack-dev-server --progress --colors --open",
-    "test": "npm run build-e2e && jest",
+    "start": "yarn build-example && webpack-dev-server --progress --colors --open",
```

Comment:
Lets stick with `npm` here.

Reply:
no problem, ill rewrite the scripts for npm and re-test. i should get this done later today or tomorrow.

Reply:
I reverted the scripts back to npm and removed the yarn.lock file